### DISCUSSION TITLE
Azure AD SSO is broken

### DISCUSSION THREAD

**Discussion by foolishhare** [requarks/wiki#4704] [Error / Bug Report] [answered]
When I try to set up SSO via Azure AD I got the error, but I make all how documentation is described.
![image](https://user-images.githubusercontent.com/14123319/141980549-b88a458c-b565-4b48-a3c1-63a3c0947043.png)
![image](https://user-images.githubusercontent.com/14123319/141980651-7c32843e-99a2-4dac-9178-77f98afa4e92.png)
![image](https://user-images.githubusercontent.com/14123319/141980797-7e5fc2f5-8dd7-491c-9fe2-541200e902f3.png)
![image](https://user-images.githubusercontent.com/14123319/141980845-44b81a9b-8ebb-43b7-8310-e7e4eca2a58a.png)
![image](https://user-images.githubusercontent.com/14123319/141981182-515fe9e2-13a4-43ef-9d1f-fb461f930b3c.png)

**Comment by foolishhare** [comment]
As an error state, the problem is to redirect URI. I set up it in Azure AD correctly. I suppose problem somewhere in Wikijs

**Comment by foolishhare** [comment, marked as answer]
I find the problem 
![image](https://user-images.githubusercontent.com/14123319/142013554-633dfef1-c8ed-46d7-99fa-28476b3352c9.png)
In general settings protocol was set to HTTP not HTTPS

**Reply by ghost** [reply]
Is this the only thing you did to fix the problem? My URL already is in https. The problem still persists...
